### DISCUSSION TITLE
PBA Rush

### DISCUSSION THREAD

**Discussion by thedarkdragon11** [iptv-org/community#720] [Q&A]
**Channel Name:** PBA Rush
**Channel ID:** PBARush.ph
**Country:** Philippines
**Language:** English, Filipino
**Website:** https://trueid.ph/livetv/XpjqQjb3YNwP/pba-rush/
**Stream URL (optional):** xxx
**Notes (optional):** There's a tokenized stream in the website,

**Comment by BellezaEmporium** [comment]
POST on https://trueid.ph/cmsPostProxy/contents/video/XpjqQjb3YNwP/streamer?os=android gives the link

Blocked on every country except PH.

**Comment by thedarkdragon11** [comment]
> 
> 
> POST on https://trueid.ph/cmsPostProxy/contents/video/XpjqQjb3YNwP/streamer?os=android gives the link
> 
> Blocked on every country except PH.

So it's not possible then?

**Comment by BellezaEmporium** [comment]
It could be possible, but I shall take a look using a Filipino IP.

**Comment by thedarkdragon11** [comment]
> 
> 
> It could be possible, but I shall take a look using a Filipino IP.

I can setup a temporary VM you can access remotely if necessary/last resort.

**Comment by BellezaEmporium** [comment]
They usually do not block countries around the Phillipines. Singapore being a good example from time to time.

**Comment by thedarkdragon11** [comment]
> 
> 
> They usually do not block countries around the Phillipines. Singapore being a good example from time to time.

For testing purposes?

**Comment by BellezaEmporium** [comment]
More than this. I do not know if they have strong diplomacy ties with Singapore or Taiwan, but when using those IPs, I usually do not get blocked.

**Comment by thedarkdragon11** [comment]
> 
> 
> More than this. I do not know if they have strong diplomacy ties with Singapore or Taiwan, but when using those IPs, I usually do not get blocked.

That's quite surprising. I hope it works.

**Comment by thedarkdragon11** [comment]
> 
> 
> More than this. I do not know if they have strong diplomacy ties with Singapore or Taiwan, but when using those IPs, I usually do not get blocked.

If IPs from SG or TW won't work, I can setup a VM with a natural PH IP.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19746
